lib/dateFormatter.js

```diff
@@ -1,18 +1,27 @@
 import fecha from 'fecha'
 
+function getUtcString (timestamp) {
```

Comment:
Without this, browsers were using the current time zone, shifting the days from Thursday/Friday to Wednesday/Thursday in the US, e.g. 😱